src/ndb/ndb_weaks.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Roman Gezikov <rgezikov@exafore.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+#include "ndb.h"
+#include "ndb_internal.h"
+
+
+enum ndb_op_code ndb_gps_l2cm_l2c_cap_store(u32 *l2c_cap,
```

Comment:
I think all the read/store/update functions should have `NDB_WEAK` and the init/sbp function should not.